landoapi/app.py

```diff
@@ -111,3 +111,19 @@ def construct_app(config: dict[str, Any]) -> connexion.App:
     initialize_hooks(flask_app)
 
     return app
+
+
+def construct_treestatus_app(config: dict[str, Any]) -> connexion.App:
+    app = connexion.App(__name__, specification_dir="spec/")
+
+    app.add_api(
+        "treestatus.yml",
+        resolver=RestyResolver("landoapi.api"),
+        options={"swagger_ui": False},
+    )
```

Comment:
As mentioned elsewhere, I think it makes sense to simply add the treestatus spec to the existing app, I don't see the need to create a completely separate app instance, as this causes code duplication.

Even if you were to stay with the single API spec per app approach, you could instead just decide on which API spec to add to the app based on the command or command parameters. E.g.,

```python
def construct_app(config: dict[str, Any], spec="swagger.yml") -> connexion.App:
    app = connexion.App(__name__, specification_dir="spec/")
    app.add_api(
        spec,
        resolver=RestyResolver("landoapi.api"),
        options={"swagger_ui": False},
    )
    flask_app = app.app
    flask_app.config.update(config)
    flask_app.register_blueprint(dockerflow)
    initialize_hooks(flask_app)

    return app
```

And then you can call `constuct_app(config, "treestatus.yml")` where relevant based on the command.

Reply:
I described why we need to load the Treestatus app as a separate, non-namespaced app in my comment reply to your overall review comment. I have added the `spec` argument to `construct_app` as suggested.